tests/src/test/java/org/teavm/jso/test/JSWrapperTest.java

```diff
@@ -331,6 +332,28 @@ public void jsToString() {
         assertEquals("bar", b.toString());
     }
 
+    @Test
+    public void jsSymbol() {
```

Comment:
`JSWrapperTest` has nothing to do with `JSSymbol`. Consider creating a separate test class for `JSSymbol`